packages/toolkit/src/codewhisperer/models/model.ts

```diff
@@ -267,6 +267,10 @@ export class TransformByQState {
     private projectName: string = ''
     private projectPath: string = ''
 
+    private totalSourceCodeBytes: number = 0
```

Comment:
Is any of this code used in this PR, or just left over from another change?

Reply:
Update: removed some of this code so N/A now